ui/packages/ui/src/app/Layout/AppLayout.tsx

```diff
@@ -21,7 +21,13 @@ const AppLayout: React.FC = ({ children }) => {
 	);
 	const handleClusterIdChange = (id: number) => {
 		setClusterId(id);
-	}
+	};
+	const commitLink = `https://github.com/debezium/debezium-ui/commit/${process.env.COMMIT_HASH}`;
+	const Footer = () => (
+  	<footer class="footer">
```

Comment:
Should be `className` instead of `class`

Reply:
:doh: I forgot about that, my react/ts is a little rusty.